modules/Bio/EnsEMBL/Analysis/Hive/Config/RepeatmodelerPipeline_conf.pm

```diff
@@ -13,51 +13,67 @@ sub default_options {
   return {
     # inherit other stuff from the base class
     %{ $self->SUPER::default_options() },
+     user => $ENV{'USER_R'},
+     password => $ENV{'GBPASS'},
+     user_w => $ENV{'GBUSER'},
+     'pipeline_db' => {
+	-dbname => 'repeatmodeler_test',
+        -host   => $ENV{GBS1},
+        -port   => $ENV{GBP1},
+        -user   => $self->o('user_w'),
+         -pass   => $self->o('password'),
+	-driver => 'mysql',
+     },
```

Comment:
We are trying to move ENV variables out of the code as much as possible. A lot of these will break in the new environment set up, so best here to either leave to be filled in by passed parameters when doing init_pipeline or provide a config/registry for this information